### PR TITLE
feat: add loading state to highlight cards

### LINKED ISSUE #903: Feature: Dashboard loading should not show 0 when loading
### Type of feature

🍕 Feature

### Current behavior

There is a 25 second time to load content. We need to add some skeletons or loading indicators.

 
Screen Shot 2023-02-22 at 12 59 02 PM

### Suggested solution

Use or create a skeleton wrappers.

https://github.com/open-sauced/insights/blob/ad78edd8b4b01976188ebafffb0413476f2e1359/components/organisms/Contributors/contributors.tsx#L14

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by Deadreyo**
## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description

Adds a loading state to the Highlight Card component, and makes use of it in the Dashboard to display skeleton till the data is loaded.
## Related Tickets & Documents

Fixes #903 
## Mobile & Desktop Screenshots/Recordings

image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by brandonroberts — APPROVED** [review]

**Review by OgDev-01 — APPROVED** [review]
LGTM 🍕 . Skeleton count can be 1 with an icreased height to match cards... I think that suits better for the card

File: components/molecules/HighlightCard/highlight-card.tsx [hunk 1]
```diff
@@ -108,51 +111,55 @@ const HighlightCard: React.FC<HighlightCardProps> = ({
...
         {/* Main Information */}
-        <div className="flex flex-col w-full px-6 pb-5 mt-2">
-          {/* Main Number */}
-          <div className="flex flex-col items-center">
-            {/* Percentage */}
-            <div className="text-4xl">
-              {percentage !== undefined ? `${percentage}%` : <span></span>}{value !== undefined ? value : ""}
-            </div>
+        { isLoading ? (
+          <SkeletonWrapper height={79} count={2} />
```
Comment (OgDev-01) [inline components/molecules/HighlightCard/highlight-card.tsx:115]: Make count 1 or remove entirely, then increase height to match card

**Review by bdougie — APPROVED** [review]
This works

File: components/molecules/HighlightCard/highlight-card.tsx (hunk 1 above)
Comment (Deadreyo) [reply components/molecules/HighlightCard/highlight-card.tsx:115]: I intended to make it 2. Looked a lot nicer to me

**Review by 0-vortex — APPROVED** [review]
LGTM! 👍

**Pull request merged**